### ISSUE TITLE
Volume Init Failing

### ISSUE THREAD

**Issue by ghost** [nsayer/Orthrus#1]
I recently bought an Orthrus v3.0.3 and two 16 GB SDHC cards. When I place the cards in the device, the error light comes on as expected. Then I press and hold and the error light flashes for five seconds. After that, it shuts off and I see the activity light blink for probably less than 100 milliseconds. Then the error light comes back on.

I'd provide more info but that's kind of hard to do. Any ideas?

**Comment by nsayer** [comment]
If you see the activity light blink then that’s activity from the host, so in all likelihood the initialization succeeded. You can verify that by looking at the cards on a host - they won’t have normal partition tables any more. 

This likely indicates that high speed access is causing errors. What kind of cards are these? Can you try with two other brand cards just to see?

It’s possible that you have bad hardware too, but I’ve not seen yet a unit fail in this way.

**Comment by ghost** [comment]
You're right, the initialization is successful; I can see the OrthrusVolume string on the devices.

These cards are Strontium 85MB/s, SDHC, UHS-I U1, Class 10 cards. Believe it or not, at the moment, I only have three cards; these two 16 GB ones and a 64 GB version of the same cards. I tried one 16 GB card with the 64 GB card since it's the only other combination I have available right now but it does the same thing.

**Comment by nsayer** [comment]
Well, we can try swapping out the hardware. Can you mail me privately with your order number and we’ll make that happen?

**Comment by ghost** [comment]
@nsayer I really appreciate the offer and will definitely take you up on it. However, I'd feel bad if I didn't at least try some other SD cards or other debugging options if possible... unless such ideas are dead ends. Is there any way I can debug the device? Maybe by writing firmware to it that sends debug info back to my PC via the USB cable?

**Comment by nsayer** [comment]
There aren't any debugging provisions in the firmware over USB as-is. In development, I was debugging via SWD over that interface. You can populate that header and grab the source code and do the whole Atmel Start thing and become a firmware developer yourself. :)

As for the cards, I've had really good luck with Sandisk cards.

**Comment by ghost** [comment]
@nsayer Hmm, I'm not terribly familiar with how USB works; would it be terribly difficult to get a serial port over the USB connection?

As for cards, if you happen to have any specific Sandisk cards I should try, I'll do so. Otherwise I'll go pick out some Sandisk 8GB cards.

If nothing else, I'll get ahold if you privately.

**Comment by nsayer** [comment]
I used Sandisk 8GB extensively in development and testing. I'd be surprised if those gave you any trouble.

**Comment by ghost** [comment]
Alright, I'll give these a shot. I'd love to avoid the trouble of a hardware swap if possible. I'll be in touch.
